tests/jobs/test_scheduler_job.py

```diff
@@ -119,10 +119,18 @@ def clean_db():
         # DO NOT try to run clear_db_serialized_dags() here - this will break the tests
         # The tests expect DAGs to be fully loaded here via setUpClass method below
 
+    def setup_method(self):
+        self.clean_db()
+
+    def teardown_method(self):
+        if self.scheduler_job and self.scheduler_job.processor_agent:
+            self.scheduler_job.processor_agent.end()
+            self.scheduler_job = None
+        self.clean_db()
```

Comment:
Any particualr reason for this change? This does not seem to change things functionality-wise afaict

Reply:
I had error while trying to delete DagModel in a previous push but I can see now that here is not the problem so, I will revert it